pkg/provider/aws/knowledgebase/api_gateway.go

```diff
@@ -109,6 +109,99 @@ var ApiGatewayKB = knowledgebase.Build(
 			return nil
 		},
 	},
+	knowledgebase.EdgeBuilder[*resources.ApiIntegration, *resources.LoadBalancer]{
+		Expand: func(integration *resources.ApiIntegration, lb *resources.LoadBalancer, dag *core.ResourceGraph, data knowledgebase.EdgeData) error {
+			// This isnt an instance of an expanded path, rather an existing edge so ignore
+			if integration.Name != "" {
+				return nil
+			}
+			restApi, ok := data.Source.(*resources.RestApi)
+			refs := core.AnnotationKeySet{}
+			refs.AddAll(lb.ConstructsRef)
+			refs.AddAll(restApi.ConstructsRef)
```

Comment:
nit:
```go
lb.ConstructsRef.CloneWith(restApi.ConstructsRef)
```
This is a common pattern, so there's a special method for it.

Reply:
👍 changed it to this, good callout, need to make my brain remember these patterns now